Sources/NIO/ByteBuffer-core.swift

```diff
@@ -659,9 +659,15 @@ public struct ByteBuffer {
     ///
     /// - note: This method will allocate if the underlying storage is referenced by another `ByteBuffer`. Even if an
     ///         allocation is necessary this will be cheaper as the copy of the storage is elided.
-    public mutating func clear() {
+    ///
+    /// - parameters:
+    ///     - minimumNeededCapacity: The minimum capacity that will be (re)allocated for this buffer
+    public mutating func clear(capacity minimumNeededCapacity: _Capacity? = nil) {
```

Comment:
Combined with the comment above, this capacity may as well be non-optional and remove the default value.

Reply:
Sure! Updated!